modular_skyrat/modules/modular_implants/code/nifsofts/station_pass.dm

```diff
@@ -0,0 +1,191 @@
+/obj/item/disk/nifsoft_uploader/station_pass
+	name = "Automatic Apprasial"
+	loaded_nifsoft = /datum/nifsoft/station_pass
+
+//Modular Persistence variables for the station_pass NIFSoft
+/datum/modular_persistence
+	///What name is saved to the station pass NIFSoft?
+	var/station_pass_nifsoft_name
+	///What message is saved to the station pass NIFSoft?
+	var/station_pass_nifsoft_message
+
+/datum/nifsoft/station_pass
```

Comment:
This should be renamed to the name of the actual software, otherwise it makes little sense.